data/sql/base/db_world/item_template_locale.sql

```diff
@@ -168965,6 +168966,7 @@ INSERT INTO `item_template_locale` VALUES
 (30635,'deDE','Schlüssel der Zeit','Eröffnet den Schwierigkeitsgrad \'Heroisch\' für Dungeons der Höhlen der Zeit.',18019),
 (30636,'deDE','NPC Equip 30636','',15050),
 (30637,'deDE','Flammengeschmiedeter Schlüssel','Eröffnet den Schwierigkeitsgrad \'Heroisch\' für Dungeons der Höllenfeuerzitadelle.',18019),
+(30637,'zhCN', '焰铸钥匙', '允许你进入英雄难度的地狱火堡垒地下城。', 15050),
```

Comment:
hi, thanks for your contributions. However, we don't modify the base files directly like this, please read this how-to:

https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
I tried, should be good now.